lexer/lexer.go

```diff
@@ -108,6 +121,24 @@ func (l *Lexer) NextToken() token.Token {
 			if strings.ToUpper(tok.Literal) == token.ASC {
 				return token.Token{Type: token.ASC, Literal: token.ASC}
 			}
+			if strings.ToUpper(tok.Literal) == token.TRUE {
+				return token.Token{Type: token.BOOL, Literal: token.TRUE}
+			}
+			if strings.ToUpper(tok.Literal) == token.FALSE {
+				return token.Token{Type: token.BOOL, Literal: token.FALSE}
+			}
+			if strings.ToUpper(tok.Literal) == token.AND {
+				return token.Token{Type: token.AND, Literal: token.AND}
+			}
+			if strings.ToUpper(tok.Literal) == token.OR {
+				return token.Token{Type: token.OR, Literal: token.OR}
+			}
+			if strings.ToUpper(tok.Literal) == token.BOOL {
+				return token.Token{Type: token.BOOLEAN, Literal: token.BOOLEAN}
+			}
+			if strings.ToUpper(tok.Literal) == token.BOOLEAN {
+				return token.Token{Type: token.BOOLEAN, Literal: token.BOOLEAN}
+			}
```

Comment:
This list is stupidly long now, I should soon refactor this to loop over a map or slice of tokens instead.